x-pack/plugins/ingest_pipelines/__jest__/client_integration/ingest_pipelines_filter_list.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getPipelinesMock,
+  getFilteredPipelinesMock,
+  getSelectedPipelinesMock,
+} from './helpers/pipelines_filter_list.helpers';
+
+describe('Filtered Pipelines', () => {
+  const pipelines = getPipelinesMock();
+
+  test('should display all pipelines', () => {
+    expect(getFilteredPipelinesMock(pipelines, ['on', 'on'])).toEqual(pipelines);
+  });
+
+  test('should display only managed pipelines', () => {
+    const expectedResult = getSelectedPipelinesMock([0, 1]);
+
+    expect(getFilteredPipelinesMock(pipelines, ['on', 'off'])).toEqual(expectedResult);
```

Comment:
It looks like we're just testing mocks. Is it possible to create tests that simulate clicking the filter button and verify the items shown in the table?